api/oh.yaml

```diff
@@ -8,8 +8,8 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0
-host: "oh2.open-hospital.org"
-basePath: /oh-api
+host: "localhost:8080"
```

Comment:
is this change meant to be? for me would very good but... you find the way to customize this parameter for different env (develop, test, production, etc...)?